Source/Core/Matrix4.js

```diff
@@ -1818,13 +1820,13 @@ Matrix4.multiplyByTranslation = function (matrix, translation, result) {
 var uniformScaleScratch = new Cartesian3();
 
 /**
- * Multiplies an affine transformation matrix (with a bottom row of <code>[0.0, 0.0, 0.0, 1.0]</code>)
+ * Multiplies a proper rigid transformation matrix (with a bottom row of <code>[0.0, 0.0, 0.0, 1.0]</code>)
```

Comment:
Like the method below, this only applies a uniform scale to the 3x3 portion of an affine transform.